engine/flink/management/dev-model/src/test/resources/extractedTypes/devCreator.json

```diff
@@ -24,6 +24,23 @@
     },
     "staticMethods": {}
   },
+  {
+    "clazzName": {
+      "type" : "Unknown"
```

Comment:
Shoulndn't something change near `refClazzName` usages on FE side? I see that it isn't optional